js/components/TestEventsData.js

```diff
@@ -0,0 +1,384 @@
+export const testEventData = [
```

Comment:
This is just for testing purpose, you can modify the types / working groups of events data, and pass the test data into Events component prop to see the dynamically hide/show filters. Will remove from here afterwards and move it into tests.